lib/scholar/naive_bayes/complement.ex

```diff
@@ -0,0 +1,440 @@
+defmodule Scholar.NaiveBayes.Complement do
+  @moduledoc """
+  The Complement Naive Bayes classifier described in Rennie et al. (2003).
+
+  The Complement Naive Bayes classifier was designed to correct the “severe assumptions” made
```

Comment:
I think it'd be good to either add a reference or expand directly on what the sever assumptions are